frame/composable-traits/src/vault.rs

```diff
@@ -23,6 +23,8 @@ pub enum FundsAvailability<Balance> {
 	/// Example, the strategy was removed by the fund manager or governance, so all funds should
 	/// be returned.
 	MustLiquidate,
+	/// When there are no balance that can be withdrawn or deposit and don't need to be liquidated
+	Equilibrable,
```

Comment:
Tests for `vault` pallet have failed. Needed to cover `Equilibrable` match arm.

Reply:
Sure! Thanks for it. Resolved!